cmd/ctr/commands/images/export.go

```diff
@@ -69,6 +77,66 @@ When '--all-platforms' is given all images in a manifest list must be available.
 			return errors.New("please provide both an output filename and an image reference to export")
 		}
 
+		client, ctx, cancel, err := commands.NewClient(context)
+		if err != nil {
+			return err
+		}
+		defer cancel()
+
+		var w io.WriteCloser
+		if out == "-" {
+			w = os.Stdout
+		} else {
+			w, err = os.Create(out)
+			if err != nil {
+				return err
+			}
+		}
+		defer w.Close()
+
+		if !context.BoolT("local") {
+			pf, done := ProgressHandler(ctx, os.Stdout)
+			defer done()
+
+			exportOpts := []tarchive.ExportOpt{}
+			if pss := context.StringSlice("platform"); len(pss) > 0 {
+				for _, ps := range pss {
+					p, err := platforms.Parse(ps)
+					if err != nil {
+						return fmt.Errorf("invalid platform %q: %w", ps, err)
+					}
+					exportOpts = append(exportOpts, tarchive.WithPlatform(p))
+				}
+			}
+			if context.Bool("all-platforms") {
+				exportOpts = append(exportOpts, tarchive.WithAllPlatforms)
+			}
+
+			if context.Bool("skip-manifest-json") {
+				exportOpts = append(exportOpts, tarchive.WithSkipCompatibilityManifest)
+			}
+
+			if context.Bool("skip-non-distributable") {
+				exportOpts = append(exportOpts, tarchive.WithSkipNonDistributableBlobs)
+			}
+
+			storeOpts := []image.StoreOpt{}
+			for _, img := range images {
+				storeOpts = append(storeOpts, image.WithExtraReference(img))
+			}
+
+			err := client.Transfer(ctx,
+				image.NewStore("", storeOpts...),
+				tarchive.NewImageExportStream(w, "", exportOpts...),
+				transfer.WithProgress(pf),
+			)
+			if err != nil {
+				return err
+			}
+
+			return nil
```

Comment:
nit ... without the progress update here.. return err would be the same as !=nil return.. then return nil here